dapper/cmd/dapper-db-meta-ingest/main.go

```diff
@@ -172,43 +172,45 @@ func main() {
 				}
 			}
 
-			for _, l := range km.Relations {
+			for toKey, rs := range km.Relations {
 				// Makes sure if the keys exists
 				found := false
 				for _, k := range input.Metadata {
-					if l.Relation.FromKey == k.Key {
+					if km.Key == k.Key {
 						found = true
 					}
 				}
 
 				if !found {
-					tempErr := fmt.Errorf("FromKey %s/%s not found in metadata", l.Relation.Domain, l.Relation.FromKey)
+					tempErr := fmt.Errorf("FromKey %s/%s not found in metadata", km.Domain, km.Key)
 					log.Println(tempErr)
 					txErr = tempErr
 					return
 				}
 
 				found = false
 				for _, k := range input.Metadata {
-					if l.Relation.ToKey == k.Key {
+					if toKey == k.Key {
 						found = true
 					}
 				}
 
 				if !found {
-					tempErr := fmt.Errorf("ToKey %s/%s not found in metadata", l.Relation.Domain, l.Relation.ToKey)
+					tempErr := fmt.Errorf("ToKey %s/%s not found in metadata", km.Domain, toKey)
 					log.Println(tempErr)
 					txErr = tempErr
 					return
```

Comment:
shall we `continue` to check next `toKey`?

Reply:
There's something wrong with this protobuf file because this shouldn't happen. So we're rejecting the whole file.